solutions.txt

```diff
@@ -192,8 +181,7 @@ let numList=[]
 numList= numArrays[numArrays.length-1];
 // to return the last nested array. 
 
-console.log('Exercise 13 Result:
-', numList);
+console.log('Exercise 13 Result:', numList);
```

Comment:
I can't leave a comment closer to the function you wrote, but about fizzbuzz:

you always want to put the most specific condition first. Otherwise, any number divisible by both 3 and 5 will be caught by the first condition it meets (in this case, being divisible by 3). lmk if you have any questions about that. Otherwise, your code looks good!